docs/modules/ROOT/pages/guides/v4-migration/authorization.adoc

```diff
@@ -0,0 +1,272 @@
+
+=== `bind`
+
+Given an `bind` rule, which checks the `id` field of a `User` against the JWT subject _after_ any operation:
+
+[source, graphql, indent=0]
+----
+type User @auth(rules: [{ bind: { id: "$jwt.sub" } }]) {
+    id: ID!
+}
+----
+
+This will now become:
+
+[source, graphql, indent=0]
+----
+type User @authorization(validate: [{ when: [AFTER], where: { node: { id: "$jwt.sub" } } }]) {
+    id: ID!
+}
+----
+
+Note that `bind` is no longer a discrete rule, but configured by a `when` argument which is an array accepting values `BEFORE` and `AFTER`.
+
+It is our view that users will quite rarely need to specify this argument as it defaults to both, and users will often want to validate a node property both before and after each operation.
+
+=== `isAuthenticated`
+
+WARNING: There isn't a direct replacement for the `isAuthenticated` argument, as you will read in this section. Please https://github.com/neo4j/graphql/issues/new/choose[raise a feature request] if this is an absolute blocker to migration.
+
+Given a previous type definition, which required authentication for any operation on the type `User`:
+
+[source, graphql, indent=0]
+----
+type User @auth(rules: [{ isAuthenticated: true }]) {
+    id: ID!
+}
+----
+
+There is not a rule under `@authorization` for this any more, but the closest is:
+
+[source, graphql, indent=0]
+----
+type User @authentication {
+    id: ID!
+}
+----
+
+The difference here being that for example, given the following query:
+
+[source, graphql, indent=0]
+----
+{
+    users(where: { id: "1" }) {
+        id
+    }
+}
+----
+
+* `@auth(rules: [{ isAuthenticated: true }])` would only throw an error if the `where: { id: "1" }` filter resulted in a match on a `User`.
+* `@authentication` will always throw an error if a user is not authenticated, and this will happen _before_ database execution to restrict database access to queries generated by authenticated users only.
+
+=== `roles`
+
+[NOTE] 
+====
+For these examples, the following type is required in the type definitions:
+
+[source, graphql, indent=0]
+----
+type JWT @jwt {
+    roles: [String!]!
+}
+----
+====
+
+Given the following type definition, which required a user to have the "admin" role to perform any operation on the type `User`:
+
+[source, graphql, indent=0]
+----
+type User @auth(rules: [{ roles: "admin" }]) {
+    id: ID!
+}
+----
+
+This will now become:
+
+[source, graphql, indent=0]
+----
+type User @authorization(validate: [{ where: { jwt: { roles_INCLUDES: "admin" } } }]) {
+    id: ID!
+}
+----
+
+Note the following:
+
+* This is inside a `validate` rule, which will throw an error without the role as per the previous implementation. This can also be used inside a `filter` rule to just return zero results if a user does not have the required role.
+* `roles` has become `roles_INCLUDES`, because the xref::filtering.adoc[full filtering capabilities of the library] can now be used within the `@authorization` directive.
```

Comment:
```suggestion
* `roles` has become `roles_INCLUDES`, because the xref::reference/filtering.adoc[full filtering capabilities of the library] can now be used within the `@authorization` directive.
```